src/backend/lib/elastic.js

```diff
@@ -30,9 +30,20 @@ const useMockElastic = process.env.MOCK_ELASTIC;
 // Use either a real Elastic Client or a Mock instance, depending on env setting
 const ElasticConstructor = useMockElastic ? MockClient : Client;
 
+// parsesUrl to make sure there is only a single port
+const parseUrl = (url, port) => {
+  if (url) {
+    const urlObj = new URL(url);
+
+    return urlObj.port === '' ? `${url}:${port}` : url;
+  }
+
+  return null;
+};
+
 function createElasticClient() {
   try {
-    const elasticUrl = `${ELASTIC_URL}:${ELASTIC_PORT}` || 'http://localhost:9200';
+    const elasticUrl = parseUrl(ELASTIC_URL, ELASTIC_PORT) || 'http://localhost:9200';
```

Comment:
Take a look at what we do to parse [redis' url](https://github.com/Seneca-CDOT/telescope/blob/7531ea5bbb3eb9f40d16b744e418ead6ca9afb69/src/backend/lib/redis.js#L18), you might be able to do something similar here.

Reply:
@manekenpix  is that not deprecated now. I believe node URL has deprecated it. https://nodejs.org/api/url.html

Reply:
Yeah, we can switch to the global `URL` constructor everywhere.  @zg3d want to fix that for Redis while you're doing this surgery?  Also, check if we do it anywhere else.

Reply:
@zg3d I completely forgot :+1:

Reply:
Ya, I can do that.